src/mongo-service.js

```diff
@@ -10,15 +10,16 @@ const defaultOptions = {
   addUpdatedOnField: true,
   useStringId: true,
   validate: undefined,
+  emitter: new EventEmitter(),
 };
 
 class MongoService extends MongoQueryService {
-  constructor(collection, options = {}, eventBus = new EventEmitter()) {
+  constructor(collection, options = {}) {
```

Comment:
@NesterenkoNikita 
Is there any reason why it not backward compatible with previous versions?

Reply:
Nope. Actually, it is not a braking change, because we have not mentioned the third parameter anywhere in the docs before. 🙃